mocha/generic-config.json

```diff
@@ -2,8 +2,9 @@
     "AUTH_REQUEST": {
         "client_id": "${client-id}",
         "response_type": "code",
-        "scope": "openid email",
-        "redirect_uri": "https://${domain-name}${callback-path}"
+        "scope": "${scope}",
+        "redirect_uri": "https://${domain-name}${callback-path}",
+        "idp": "${idp}"
```

Comment:
`idp` defaults to an empty string. Is it valid to pass an empty string for this parameter?

Reply:
Yes empty string is valid.